server/src/test/scala/org/http4s/server/middleware/TimeoutSpec.scala

```diff
@@ -48,12 +48,7 @@ class TimeoutSpec extends Http4sSpec {
       var clean = new AtomicBoolean(false)
       val service = HttpService {
         case _ =>
-          implicit val A = Task.asyncInstance(Strategy.sequential)
-          (sleep_(200.milliseconds) ++
-            eval(NoContent()) ++
-            eval_(Task.delay(clean.set(true))))
-            .runLast
-            .map(_.getOrElse(throw new AssertionError("Should have emitted NoContent")))
+          NoContent().schedule(2.seconds) <* Task.delay(clean.set(true))
```

Comment:
Is `<*` commonly understood?  Guessing so.

Unfortunately no scaladoc to explain it:
http://typelevel.org/cats/api/cats/syntax/CartesianOps.html#%3C*[B](fb:F[B])(implicitF:cats.Functor[F]):F[A]

To be fair, I have a much better shot at understanding the version in green than the one in red. :smile: